README.md

```diff
@@ -28,7 +28,27 @@ export default class Demo extends Component {
         return (
             <AutoHeightImage
                 width={100}
-                imageURL={'http://placehold.it/350x150'}
+                image={'http://placehold.it/350x150'}
```

Comment:
`image={'http://placehold.it/350x150'}` should be `source={{ uri: 'http://placehold.it/350x150' }}`